docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing guide for Pokémon HeartGold
+
+This repo contains binary code extracted from Pokémon - HeartGold Version (USA), disassembled into .s files.
+The goal is to take the assembly code and turn them into readable C code that generates the same assembly instructions.
+
+## Prerequisites
+
+* Basic knowledge of ARMv4T assembly.
```

Comment:
```suggestion
* Basic knowledge of ARMv5TE assembly.
```

Reply:
had GBA on my mind, whoops! this is the correct one.